azure-pipelines.yml

```diff
@@ -247,9 +249,17 @@ jobs:
       GCC6:
         ConfigOpts: --compiler=g++-6 --xerces --features=uses_wchar=1 --no-debug
         PackageDeps: libxerces-c-dev g++-6
+      GCC8:
+        ConfigOpts: --compiler=g++-8 --xerces --no-inline --java=$(JAVA_HOME_12_X64)
+        TEST_JAVA_MESSENGER: true
```

Comment:
We should probably run the JAVA messenger tests for the other JAVA builds as well (Debug & Release)

Reply:
Done.